src/template.html

```diff
@@ -14,7 +14,7 @@
         <h3>Today's To Do <i class="fa-solid fa-arrows-rotate"></i></h3>
         <input type="text" class="input-field" placeholder="Add to your list..." />
         <ul class="tasks"></ul>
-        <button type="reset">Clear all completed</button>
+        <button type="reset">Clear all completed.</button>
```

Comment:
- [ ] Your new functionalities should be on a new file, kindly move them from `index.js`

Reply:
@Ismailco all is done i seperated my functionalities